.changelog

```diff
@@ -4,6 +4,7 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)
 
+- Fix RHEL channel names for bootstrapping
```

Comment:
```suggestion
- Fixed RHEL channel names for bootstrapping in Client Configuration Guide
```
We sometimes use the RHEL abbrev here, sometimes the expanded variant.  I do not know whether consistency is needed in this case.